packages/libheif.rb

```diff
@@ -23,32 +23,29 @@ class Libheif < Package
      x86_64: '626d99ad78adb2e959bc0cf812ce53da110d79d3917378b110b914f545137660'
   })
 
```

Comment:
Update to https://github.com/strukturag/libheif/releases/download/v1.15.1/libheif-1.15.1.tar.gz?